src/main/java/edu/ucsb/cs156/courses/repositories/GradeRepository.java

```diff
@@ -0,0 +1,15 @@
+package edu.ucsb.cs156.courses.repositories;
+
+import edu.ucsb.cs156.courses.entities.Grade;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+
+@Repository
+public interface GradeRepository extends CrudRepository<Grade, Long> {
+//   Optional<Grade> findbyCourseName(String courseName);
+//   Optional<Grade> findbyCourseNameAndQuarter(String courseName, String quarter);
```

Comment:
Let's leave these out until they are needed.  We try to avoid putting commented out code into the main branch.

Reply:
Okay. Fixed it!